### PR TITLE
add additional information for conTimeout value for 0 and -1

### PR DESCRIPTION
**Pull request by jmstephensgit**
Added more information to conTimeout values of 0 and -1.

New metatype is

connTimeout.desc=Amount of time after which a connection request times out. A value of -1 disables this timeout meaning infinite wait.  A value of 0 is immediate meaning no wait.

### CODE REVIEW

File: dev/com.ibm.ws.jca.cm/resources/OSGI-INF/l10n/metatype.properties [hunk 1]
```diff
@@ -29,7 +29,7 @@ agedTimeout=Aged timeout
 agedTimeout.desc=Amount of time before a connection can be discarded by pool maintenance. A value of -1 disables this timeout. A value of 0 discards every connection, which disables connection pooling.
 
 connTimeout=Connection timeout
-connTimeout.desc=Amount of time after which a connection request times out. A value of -1 disables this timeout.
+connTimeout.desc=Amount of time after which a connection request times out. A value of -1 disables this timeout meaning infinite wait.  A value of 0 is immediate meaning no wait.
```
Comment (helyarp) [inline dev/com.ibm.ws.jca.cm/resources/OSGI-INF/l10n/metatype.properties]: Suggested change: "Amount of time after which a connection request times out. A value of -1 disables this timeout, meaning infinite wait. A value of 0 is immediate, meaning no wait."

Add a comma before "meaning" in both places and remove the extra space before the last sentence.

Comment (jmstephensgit) [reply dev/com.ibm.ws.jca.cm/resources/OSGI-INF/l10n/metatype.properties]: Updated

**Comment by jmstephensgit** [comment]
#build

**Comment by jmstephensgit** [comment]
#build

**Comment by jmstephensgit** [comment]
#run-libby-bot

**Pull request merged**